lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1280,7 +1281,15 @@ public function DeleteClause()
             $this->match(Lexer::T_AS);
         }
 
-        $aliasIdentificationVariable = $this->AliasIdentificationVariable();
+        try {
```

Comment:
Maybe we should a have a comment here to explain the 💩 and specially mention that we should raise a deprecation notice in `v2.7.0`